README.rst

```diff
@@ -44,7 +44,7 @@ To run Django's test suite:
 
 * Follow the instructions in the "Unit tests" section of
   ``docs/internals/contributing/writing-code/unit-tests.txt``, published online at
-  https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/unit-tests/#running-the-unit-tests
+  https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/unit-tests/#running-the-unit-tests/
```

Comment:
Adding a trailing slash here _breaks_ the anchor link (for me) where the previous version works.